src/app/league/models.py

```diff
@@ -33,6 +34,8 @@ def update_participant_count(self, size):
 
 
 def create_league(user, name, rating_scheme, description=None):
+    if rating_scheme not in LeagueModel.rating_scheme._choices:
```

Comment:
variables beginning with an underscore are meant to be private in python (but python doesnt explicitly enforce this). Make a class variable (basically a constant within the LeagueModel class) that defines the possible choices more clearly.